challenge.js

```diff
@@ -42,7 +44,8 @@
  */
 
 // ANSWER
-
+var Dpoint = A:lastchild:
+var Epoint = Dpoint:onlyChild
```

Comment:
Very interesting answer. Sadly, CSS selectors' intersection with JS is merely being passed as strings to specific browser-provided functions. Your answer contains the necessary logic to *uniquely* select E, which was actually not required by the question. Here's an explanation of how to write it as a CSS selector:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

Your solution would look like this:

`body > :last-child > :only-child`

1